CODEOWNERS

```diff
@@ -1 +1 @@
-* @alexhung @chb0github
+* @alexhung @chb0github @danielmkn
```

Comment:
Nice try

Reply:
what do you mean?

Reply:
You added yourself as an owner

Reply:
I'm ok with adding @danielmkn to the `CODEOWNERS` list. Being a code owner means you take on the responsibility of maintaining high quality code, reviewing every PRs, ensure technical debt is kept to zero (or min).

@chb0github and I have accumulated many years of working with good and bad code bases, thus allowing us to leverage those experience when writing and reviewing code. These aren't something one can learn from book/blog/StackOverflow/etc. but everyone has to start somewhere.